src/viper/trans.ml

```diff
@@ -958,6 +979,7 @@ and typed_field t =
   | T.Prim T.Int  -> "$int"
   | T.Prim T.Nat  -> "$int"  (* Viper has no native support for Nat, so translate to Int *)
   | T.Prim T.Bool -> "$bool"
+  | T.Prim T.Text -> "$int"  (* Viper has no native support for Text, so translate to uninterpreted Int values *)
```

Comment:
```suggestion
  | T.Prim T.Text -> "$text"
```
+ `field $text: Int` into prelude
I think it will make generated code a bit more readable.

Reply:
Done